### ISSUE TITLE
add support for windows containers(Windows 2016 Docker Host)

### ISSUE THREAD

**Issue by lmayorga1980** [inspec/inspec#3457]
![screenshot from 2018-10-01 12-51-30](https://user-images.githubusercontent.com/3172378/46303033-c19f7380-c578-11e8-8f28-671fa3770254.png)

**Comment by chris-rock** [comment]
Thank you for reporting this @lmayorga1980 It would be indeed nice to improve docker support on windows

**Comment by lmayorga1980** [comment]
I will try the new docker-api branch.

https://github.com/swipely/docker-api/issues/538

**Comment by lmayorga1980** [comment]
Not sure if its a good a idea to use this approach since some dependencies still not ready for windows

`inspec exec test.rb -t docker://container_id`

or just expose the container through `winrm`

`inspec exec test.rb -t winrm://Administrator@windowshost --password 'your-password'`

**Comment by zenspider** [comment]
This is possibly fixed upstream.

**Comment by Geogboe** [comment]
Can we keep this issue open? I'd love to be able to run inspec on Windows docker images if only to use a way of testing my inspec profiles. I get the same problem as the original documented issue. I'm running:

```
> docker version
Client: Docker Engine - Community
 Cloud integration: 0.1.22
 Version:           20.10.0-beta1
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        ac365d7
 Built:             Tue Oct 13 18:13:24 2020
 OS/Arch:           windows/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.0-beta1
  API version:      1.41 (minimum version 1.24)
  Go version:       go1.13.15
  Git commit:       9c15e82
  Built:            Tue Oct 13 18:17:06 2020
  OS/Arch:          windows/amd64
  Experimental:     true

> inspec version
4.23.11
```
I'm happy to do any testing or debugging to assist with this!

**Comment by clintoncwolfe** [comment]
This is currently being worked on: https://github.com/inspec/train/issues/672

**Comment by clintoncwolfe** [comment]
This was released in inspec 4.31.0
